### ISSUE TITLE
Search的@on-result-click事件触发不了

### ISSUE THREAD

**Issue by nanliu666** [airyland/vux#3320]
<search
      @on-change="getResult"
      position="absolute"
      auto-scroll-to-top
      top="46px"
      ref="search"
      @on-result-click="resultClick"
      @on-cancel="searchCancel"
    ></search>

//js
    resultClick(item) {
      console.log(item);
      console.log(1)
    },

**Comment by Jennifer-F** [comment]
Are you resolved this problem? I have the same question.

**Comment by nanliu666** [comment]
no, I use @on-focus="getFocus" to replace @on-result-click="resultClick"

**Comment by annieh12** [comment]
> no, I use @on-focus="getFocus" to replace @on-result-click="resultClick"

@on-focus="getFocus"不是只能聚焦输入框而已嘛？跟点击结果条目差很远哇~
